wallet-new/server/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -103,6 +110,11 @@ legacyWalletBackend WalletBackendParams {..} =
               LegacyServer.walletServer (V0.convertHandler ctx) diffusion walletRunMode
       return $ withMiddleware walletRunMode app
 
+    exceptionHandler :: SomeException -> Response
+    exceptionHandler _ =
+        responseLBS badRequest400 [(hContentType, "application/json")] .
+            encode . toJSON $ GenericError "Something went wrong."
```

Comment:
`toJSON` is redundant here -- `encode :: ToJSON a => a -> LByteString` :)

Reply:
Good catch!